src/index.js

```diff
@@ -17,33 +18,12 @@ export default function instantMeiliSearch(hostUrl, apiKey, options = {}) {
         facetsDistribution: params.facets.length ? params.facets : undefined,
         facetFilters: params.facetFilters,
         attributesToHighlight: this.attributesToHighlight,
+        attributesToCrop: params.attributesToSnippet,
```

Comment:
Same in this general area. Consider deconstructing. It makes the code more readable